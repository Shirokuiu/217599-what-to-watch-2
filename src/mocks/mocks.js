export const films = [
  {
    img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    title: `Fantastic Beasts`
  },
  {
    img: `img/bohemian-rhapsody.jpg`,
    title: `Bohemian Rhapsody`
  },
  {
    img: `img/macbeth.jpg`,
    title: `Macbeth`
  },
  {
    img: `img/aviator.jpg`,
    title: `Aviator`
  },
  {
    img: `img/we-need-to-talk-about-kevin.jpg`,
    title: `We need to talk about Kevin`
  }
];
