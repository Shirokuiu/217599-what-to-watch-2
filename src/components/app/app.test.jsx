import React from "react";
import renderer from "react-test-renderer";

import App from "./app";

import {films} from "../../mocks/mocks";

it(`Is App render`, () => {
  const tree = renderer
    .create(<App
      films={films}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
