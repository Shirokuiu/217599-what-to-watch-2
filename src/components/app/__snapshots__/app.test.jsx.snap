// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is App render 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(MainPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(LoginPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/login"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(WithPrivateRoute)",
          "type": [Function],
        }
      }
      path="/movie/:movieId/add-review"
    />
    <Route
      component={[Function]}
      path="/movie/:movieId/watch"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(WithSessionStorage)",
          "type": [Function],
        }
      }
      path="/movie/:movieId"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(WithPrivateRoute)",
          "type": [Function],
        }
      }
      exact={true}
      path="/my-list"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
